receiver/k8sobjectsreceiver/unstructured_to_logdata.go

```diff
@@ -9,7 +9,7 @@ import (
 
 	"go.opentelemetry.io/collector/pdata/pcommon"
 	"go.opentelemetry.io/collector/pdata/plog"
-	semconv "go.opentelemetry.io/collector/semconv/v1.9.0"
+	semconv "go.opentelemetry.io/collector/semconv/v1.13.0"
```

Comment:
Is this upgraded needed in this PR? If not, I suggest we do it in a standalone PR.

Reply:
just checked and seems like this was not necessary - I reverted that change now